.pre-commit-config.yaml

```diff
@@ -72,12 +72,12 @@ repos:
           - --no-progress
           - --dot
           - --no-must-find-files
+          - --show-suggestions
           - --no-summary
-          - --gitignore
```

Comment:
Forgot to mention it: this is removed because it is already activated in `cspell.config.yml` via 

```yml
useGitignore: true
```